utils/formatDate.ts

```diff
@@ -44,19 +47,17 @@ export function parseApiGatewayDate(dateString: ApiGatewayDateString): Date {
 
 /**
  * Return a string that matches the API gateway format for datetimes.
- */
-export function formatFromApiGateway(date: Date): string {
-  return format(date, apiGatewayFormat)
-}
-
-/**
+ *
  * Create a Date object that is offset from today.
+ *
+ * Note: This returns a date at either midnight or 1am, depending on whether it is
+ * currently PST (-8) or PDT (-7).
  */
-export function getDateWithOffset(daysOffset = 1): Date {
+export function formatFromApiGateway(daysOffset = 1): string {
```

Comment:
Note: I eliminated `getDateWithOffset()` as a standalone function because it tempted me into falling into the trap where running tests on different machines (with different server timezones) caused tests to fail. We should always be testing using the timezone-free datetime strings that the API gateway will give us.